tests/test_plotImageSubtractionCutouts.py

```diff
@@ -365,12 +368,12 @@ def test_main_args_limit_offset(self):
             self.outputPath,
         ]
         with unittest.mock.patch.object(
-            zooniverseCutouts.ZooniverseCutoutsTask, "write_images", autospec=True,
+            plotImageSubtractionCutouts.PlotImageSubtractionCutoutsTask, "write_images", autospec=True,
```

Comment:
I didn't check in detail but this line might be getting pretty long; consider bumping part of it down.

Reply:
None of these lines were over 110, but reflowing them to have a newline for each arg/kwarg isn't a bad idea.